easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.4rc1-GCC-11.3.0.eb

```diff
@@ -0,0 +1,44 @@
+name = 'OpenMPI'
+version = '4.1.4rc1'
```

Comment:
@jfgrimm OpenMPI 4.1.4 release is there now https://www.open-mpi.org/software/ompi/v4.1/

Reply:
:+1: